src/views/Home.js

```diff
@@ -50,27 +50,44 @@ export class Home extends Component {
         );
     }
 
+    _reloadComponent() {
+        this.props.setAuthToken(undefined);
+        this._renderButtons()
+    }
+
     _renderButtons () {
         if (!this.props.token) {
             return (
                 <View>
+                    <Text style={styles.titleText}>Welcome to Eventador</Text>
+                    <Text style={styles.baseText}>Let's find you something to do tonight</Text>
+                    <Text style={styles.baseText}>{'\n\n\n'}</Text>
                     <TouchableOpacity
                         onPress={this._googleSignIn}
                         activeOpacity={0.6}>
                         <Image
-                            style={{ width: 386, height: 96 }}
-                            source={{ uri:  'http://www.setyourowntests.com/_/rsrc/1468869481521/help/accounts/btn_google_signin_dark_normal_web%402x.png' }} />
+                            style={{ width: 350, height: 45 }}
+                            source={{ uri: 'https://glow-guides.disciplemedia.com/assets/google_login_button-6474359c9c06fef79ddc3d744342cca87e0b3ef8be1d0d7b8d71c22347c84075.png' }} />
                     </TouchableOpacity>
                 </View>
             );
         } else {
             return (
-                <TouchableOpacity
-                    style={styles.viewEvents}
-                    activeOpacity={0.6}
-                    onPress={() => this.props.navigation.navigate('EventList')}>
-                    <Text style={styles.viewEventsText}>Discover Events</Text>
-                </TouchableOpacity>
+                <View>
```

Comment:
Would like to add "Hello, User" here.

Reply:
I'll add this with my attendee list PR

Reply:
This should actually be possible with the currently available info--if `this.props.user.user` is set, you should be able to get at the name with `this.props.user.user.given_name` and `this.props.user.user.family_name` (those are both fields we get from Google when the user logs in, and we keep the object from Google around in our redux store)